spec/pages/index.spec.js

```diff
@@ -1,24 +1,20 @@
-import { createLocalVue, mount } from '@vue/test-utils';
-import Vuex from 'vuex';
-import IndexPage from '~/pages/index';
+import { shallowMount } from '@vue/test-utils';
+import PageIndex from '~/pages/index';
 import members from '~/spec/sample-data/members';
 
-const localVue = createLocalVue();
-localVue.use(Vuex);
-
 describe('IndexPage', () => {
-  let store, wrapper;
+  let wrapper;
 
-  beforeAll(() => {
-    store = new Vuex.Store({
-      state: {
-        members: members,
+  beforeEach(() => {
+    wrapper = shallowMount(PageIndex, {
+      data() {
+        return {
+          members: members,
+        };
+      },
+      fetch() {
+        this.members = members;
       },
```

Comment:
On this test, you're already passing the sample data to `members` directly on L12. I don't think this fetch function is doing anything here.

You probably want to test is to see the page fetch the correct data.

You might want to mock the `fetch` and check if the data returned is expected.

Reference: https://www.leighhalliday.com/mock-fetch-jest